app/views/idv/by_mail/request_letter/index.html.erb

```diff
@@ -28,28 +34,30 @@
   </p>
 <% end %>
 
-<div class="margin-top-6 margin-bottom-5">
+<div class="margin-y-4">
```

Comment:
Minor: Based on the design / standards, I think this should be 5 units (2.5rem), not 4 (2rem).

```suggestion
<div class="margin-y-5">
```

Reply:
You are right! Updated.